docs/breadcrumbs/mass-deployment.md

```diff
@@ -0,0 +1,11 @@
+# Mass Deployment
+
+Breadcrumbs are volatile and need slightly more upkeep than Tokens and Canaries themselves. They are relatively short-lived, lasting only until the Canary's settings (or IP) changes. Depending on what was changed, new breadcrumbs may need to be generated and deployed.
+
+To make this easier for you, Breadcrumbs can be mass deployed by making use of read-only Flock API auth keys. This allows you to build them into your automation and deployment scripts.
+
+Read-only Flock API keys can be generated via the "[Flock API Key endpoints](http://localhost:8080/flocks-settings/flock-auth-token.html)" and setting `auth_token_type` to `read-only`.
+
+::: warning
+While read-only Flock API auth keys are limited to only viewing devices within a specific flock, it is recommended to remove these API keys from the console after specific script deployment has commenced, if this behavior is undesired. This can be done via the "[Remove Flock API key](http://localhost:8080/flocks-settings/flock-auth-token.html#remove-flock-api-key)" endpoint.
```

Comment:
> While read-only Flock API auth keys are limited to only viewing devices within a specific flock...

Maybe this can be simplified a bit? "While read-only Flock API auth keys limit which devices can be viewed to a specific flock, it is recommended that they be disable after they are used by scripts to deploy breadcrumbs. This can be done..."